src/schemata/definitions/workflow_template/action_objects/updateResource.yml

```diff
@@ -33,12 +33,55 @@ additionalProperties:
               type: object
               additionalProperties: false
               properties:
+                custom_attributes:
+                  $ref: '#/definitions/script'
+                  description: |
+                    To update a custom attribute, the script must resolve to a hash object with `name`, `type` and `value` attributes. The supported custom attribute data types are `number`, `text`, `boolean` (checkbox), `date`, `datetime`, `file`, `image`, `physicalQuantity`, `relation`, and `duration`.
```

Comment:
only `name` and `value` is needed.. I believe.. :thinking: 

type is being defined by the custom attributes definition in the system administration page..

Reply:
Nope, does not seem to work. Tested with this template

<details>
<summary>template</summary>

```yml
---
schema_version: '0.0.1'
info:
  version: '55.0.0'
  uuid: '8fc60467-f93a-4d2c-b6f3-84ec6e053496'
  title: 'Update Resource'
fields: {}
steps:
  stepOne:
    info:
      title: 'Show Alert'
    substeps:
      - primary: 'Now, look at this field!'
        expandDescription: true
        description: |
          Update all the things!
        buttons:
          - label: Click me
            key: alert-button

        behaviors:
          - when:
              manual:
                key: alert-button
            do:
              - update_resource:
                  id: 5
                  type: collection
                  attributes:
                    custom_attributes: |
                      {
                        HASH(
                          "name","myText1",
                          "type", "text",
                          "value", "with type"
                        ),
                        HASH(
                          "name","myText2",
                          "value", "no type"
                        )
                      }

flow:
  - stepOne

```

</details>

Reply:
yeah, that's because the custom attribute is not yet defined in the system administration page..

we can keep the changes in this PR as is for now, and we can revisit this again after we fully remove the support for local custom attribute..